analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<p>It's a bit difficult for me to find overlapping census tracts without significant effort. Maybe you can put some screenshots here, or find a way to make the interactive map highlight census tracts that overlap NCs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Added a picture showing some examples of census tracts overlapping NCs.</p>